mne/io/cnt/cnt.py

```diff
@@ -406,10 +413,12 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                 count = n_samps // channel_offset * chunk_size + extra_samps
                 n_chunks = count // chunk_size
                 samps = np.fromfile(fid, dtype=dtype, count=count)
+                n_chunks = len(samps) // n_channels // channel_offset
```

Comment:
The events change makes sense, but what is going on here? Was the `n_chunks` calculation above incorrect? If so the `n_chunks` line above should be removed.

Does `pytest mne/io` pass on your system with these changes?